themes/origins/styles/announcement-bar.scss

```
@@ -0,0 +1,29 @@
+#youcan-section--announcement-bar {
+  padding-inline: 10%;
+  background-color: rgb(var(--announcement-bar-background-color));
+  border-block-end: 1px solid
+    rgba(var(--announcement-bar-foreground-color), 0.08);
+  order: var(--announcement_bar_order);
+
+  .announcement {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .message,
+    .link {
+      font-size: calc(var(--font-heading-scale) * 1.3rem);
+      color: rgb(var(--announcement-bar-foreground-color));
+    }
+
+    .message {
+      padding-block: 0.6em;
+      text-align: center;
+    }
```

centering the text in the parent element will achieve the same result

```suggestion
  .announcement {
    text-align: center;

    .message,
    .link {
      font-size: calc(var(--font-heading-scale) * 1.3rem);
      color: rgb(var(--announcement-bar-foreground-color));
    }

    .message {
      padding-block: 0.6em;
    }
```

---

Did it like this for a future announcement slider but sure we can have the text centered for now only